index.js

```diff
@@ -129,7 +168,16 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
+  // i really want to do this with .map
```

Comment:
This sounds to me more like a job for `.filter()`

Reply:
ahh you're totally right!